script/download-libs.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -eu
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")"; pwd)" # Figure out where the script is running
+
+${SCRIPT_DIR}/download-ffi.sh
```

Comment:
I think this needs to be quoted, otherwise it won't succeed if there's spaces in the path:

```
"${SCRIPT_DIR}"/download-ffi.sh
```

Reply:
I just copied your use of it elsewhere - i'll fix all of them.